src/TestFramework/MSTest.Core/Resources/FrameworkMessages.Designer.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.VisualStudio.TestTools.UnitTesting {
     // class via a tool like ResGen or Visual Studio.
     // To add or remove a member, edit your .ResX file then rerun ResGen
     // with the /str option, or rebuild your VS project.
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "4.0.0.0")]
+    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "15.0.0.0")]
```

Comment:
Kindly remove this file. This is auto-generated.

Reply:
This file was already part of the project? Do you want me to remove it now?

Reply:
I meant remove the changes you made to this files as part of this PR.

Reply:
Can you clarify why you want to move these changes to another commit? The next time that someone will do a build this file will be changed and included in the commit that they will make.

Reply:
We do not make any changes to this file.It is just a placeholder. This file is auto-generated with every build. So once this PR will be pushed it(without this file), it will get auto-generated with the relevant code changes automatically. We don't commit any changes to this file.

Reply:
The changes have been reverted.

Reply:
@jayaranigarg Reverting the file seems to break the build? This file is not generated during the build?